pandas/core/frame.py

```diff
@@ -8218,7 +8218,6 @@ def blk_func(values):
                     result = result.iloc[0].rename(None)
                 return result
 
-        data = self
```

Comment:
The ``data = self`` at line 8247 could be collapsed together with the line below, so we get ``values = self.values``.

Reply:
data is used further down on L8263

Reply:
Right, missed that. LGTM.